requirements.txt

```diff
@@ -1 +1,3 @@
 # generated from manifests external_dependencies
+mock
```

Comment:
@baimont can't we replace the import with `from unittest import mock`?

Reply:
@guewen yes this is better. I adapted the commit